fabric-lifecycle-events-v1/src/main/java/net/fabricmc/fabric/api/client/event/lifecycle/v1/ClientLifecycleEvents.java

```diff
@@ -33,9 +33,9 @@ private ClientLifecycleEvents() {
 	 *
 	 * <p>This occurs while the splash screen is displayed.
 	 */
-	public static final Event<ClientLifecycleCallback> CLIENT_STARTED = EventFactory.createArrayBacked(ClientLifecycleCallback.class, callbacks -> client -> {
-		for (ClientLifecycleCallback callback : callbacks) {
-			callback.onChangeLifecycle(client);
+	public static final Event<ClientStarted> CLIENT_STARTED = EventFactory.createArrayBacked(ClientStarted.class, callbacks -> client -> {
+		for (ClientStarted callback : callbacks) {
+			callback.onClientStarted(client);
```

Comment:
`clientStopping.onClientStopping` etc sounds weird 🤔

Reply:
yea it definitely felt odd to type out. Any better ideas for name?